.gitignore

```diff
@@ -40,3 +40,5 @@ junit.xml
 .build
 .last_confs
 .saddle_history
+.vscode/
+node_modules_tmp
```

Comment:
Was this generated by something?

Reply:
we generate it and delete it when building the scenarios typescript, but if you cancel the build it never gets deleted and can be confusing if it ends up in your staging area

Reply:
Just FYI, this was due to a weird bug in having nested JavaScript projects (the top-level one and the nested scenarios code). Typescript will automatically look-up your directory tree and pull in modules from them. This was causing the scenario code to have multiple copies of the same libraries. To resolve the issue, we renamed the node_modules folder in the top-level during typescript compilation to prevent this behavior.

Reply:
.vscode still has no business here IMO